app/app.go

```diff
@@ -209,17 +209,27 @@ func (app *App) Units() ([]provision.Unit, error) {
 
 // MarshalJSON marshals the app in json format.
 func (app *App) MarshalJSON() ([]byte, error) {
+	var errMsgs []string
 	repo, _ := repository.Manager().GetRepository(app.ctx, app.Name)
 	result := make(map[string]interface{})
 	result["name"] = app.Name
 	result["platform"] = app.Platform
 	if version := app.GetPlatformVersion(); version != "latest" {
 		result["platform"] = fmt.Sprintf("%s:%s", app.Platform, version)
 	}
+
+	prov, err := app.getProvisioner()
+	if err != nil {
+		errMsgs = append(errMsgs, fmt.Sprintf("unable to get provisioner name: %+v", err))
+		return nil, err
```

Comment:
Because of this return the `errMsgs` assignment does nothing. I think it's safe to remove this return.